adapters/kosovo.js

```diff
@@ -0,0 +1,209 @@
+/**
```

Comment:
Ok, so something really weird. The map and table data do not match! I verified we're getting the proper data for the table and I think we should go with that, but can you just add a comment in the header here that as of this date the two pages do not agree? I would like to be as transparent as possible about that.

Reply:
Hm, just checked one station now (Drenas(, and seems the PM10 and PM2.5 are unequal, whilst NO2, SO2 and CO match. NO does not match, but that parameter is not included by Open AQ it seems. I make a note :-)